modules/admin_manual/pages/installation/linux_installation.adoc

```diff
@@ -1,14 +1,21 @@
 = Linux Package Manager Installation
+:apt-mark-hold-url: https://manpages.debian.org/stretch/apt/apt-mark.8.en.html#PREVENT_CHANGES_FOR_A_PACKAGE
 
 NOTE: Package managers should only be used for single-server setups. For production environments, we recommend installing from
 https://owncloud.org/download/#owncloud-server-tar-ball[the tar archive].
 
 [[available-packages]]
 == Available Packages
 
-The recommended package to use is `owncloud-files`. It only installs
-ownCloud, and does not install Apache, a database, or any of the
-required PHP dependencies.
+The recommended package to use is `owncloud-files`. 
+It only installs ownCloud, and does not install Apache, a database, or any of the required PHP dependencies.
+
+[TIP]
+====
+We *strongly* recommend that you {apt-mark-hold-url}[mark the `owncloud-files` package as held], by running `apt-mark hold owncloud-files`.
+This is because ownCloud can be unintentionally upgraded when upgrading other packages, such as when running `apt-get update && apt-get upgrade`. 
+Marking `owncloud-files` as "held" helps to prevent this from happening.
+====
```

Comment:
I would add a note that if you want to upgrade, you have to `mark unhold owncloud-files`, do the upgrade and if done `mark hold owncloud-files` again.

Reply:
I'm not sure that that's required, as a link is available to the relevant section of the apt-mark command.

Reply:
This is true, but it eases thinking of admins :smile:

Reply:
@mmattel - unmark/mark can be put into the section "upgrade with packages" - otherwise I doubt it being useful in the current context